frontend/src/schemas/SchemaList.tsx

```diff
@@ -1,24 +1,65 @@
-import { Button, Card, List, ListItem, ListItemText, Stack, Typography } from '@mui/material'
+import { Button, Card, Link, List, ListItem, ListItemButton, ListItemText, Stack, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { useListModels } from 'actions/model'
+import { useGetReviewRequestsForUser } from 'actions/review'
 import { deleteSchema, patchSchema, useGetSchemas } from 'actions/schema'
-import { useCallback, useMemo, useState } from 'react'
+import { useCallback, useEffect, useMemo, useState } from 'react'
 import ConfirmationDialogue from 'src/common/ConfirmationDialogue'
 import EmptyBlob from 'src/common/EmptyBlob'
 import Loading from 'src/common/Loading'
 import MessageAlert from 'src/MessageAlert'
-import { SchemaInterface, SchemaKindKeys } from 'types/types'
+import { EntryKind, SchemaInterface, SchemaKind, SchemaKindKeys } from 'types/types'
 import { getErrorMessage } from 'utils/fetcher'
-import { camelCaseToTitleCase } from 'utils/stringUtils'
+import { camelCaseToSentenceCase, camelCaseToTitleCase } from 'utils/stringUtils'
 
 interface SchemaDisplayProps {
   schemaKind: SchemaKindKeys
 }
 
+interface ObjectsToDelete {
+  primary: string
+  secondary: string
+  link: string
+}
+
 export default function SchemaList({ schemaKind }: SchemaDisplayProps) {
   const { schemas, isSchemasLoading, isSchemasError, mutateSchemas } = useGetSchemas(schemaKind)
+  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
+
+  const theme = useTheme()
 
   const [errorMessage, setErrorMessage] = useState('')
   const [open, setOpen] = useState(false)
   const [schemaToBeDeleted, setSchemaToBeDeleted] = useState('')
+  const { models, isModelsLoading, isModelsError } = useListModels(
+    schemaKind === SchemaKind.MODEL ? EntryKind.MODEL : EntryKind.DATA_CARD,
```

Comment:
This is fine for now, but we'll need to figure out a better way of linking SchemaKind to EntryKind once we add more down the line. I'll make a ticket for this.